### DISCUSSION TITLE
(idea) netherite backpacks having lodestone functionality

### DISCUSSION THREAD

**Discussion by VexedToast** [BeansGalaxy/Beans-Backpacks-2#105] [Feedback]
shift right clicking a netherite backpack with a compass would let you bind it to the backpack letting you easily find where you left it or if you died and need to find your way back (assuming no waypoint mod)

**Comment by BeansGalaxy** [comment]
That's a super cool idea. I've always wanted netherite backpacks to have something that sets them apart than just "upgrade means bigger" and this is super unique. I'll definitely be adding this.
